packages/@aws-cdk/cfnspec/spec-source/000_CloudFormationResourceSpecification.json

```diff
@@ -56641,6 +60142,52 @@
         }
       }
     },
+    "AWS::AppIntegrations::EventIntegration": {
+      "Attributes": {
+        "Associations": {
+          "ItemType": "EventIntegrationAssociation",
+          "Type": "List"
+        },
+        "EventIntegrationArn": {
+          "PrimitiveType": "String"
+        }
+      },
+      "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-appintegrations-eventintegration.html",
+      "Properties": {
+        "Description": {
+          "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-appintegrations-eventintegration.html#cfn-appintegrations-eventintegration-description",
+          "PrimitiveType": "String",
+          "Required": false,
+          "UpdateType": "Mutable"
+        },
+        "EventBridgeBus": {
+          "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-appintegrations-eventintegration.html#cfn-appintegrations-eventintegration-eventbridgebus",
+          "PrimitiveType": "String",
+          "Required": true,
+          "UpdateType": "Immutable"
+        },
+        "EventFilter": {
+          "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-appintegrations-eventintegration.html#cfn-appintegrations-eventintegration-eventfilter",
+          "PrimitiveType": "Json",
+          "Required": true,
+          "Type": "EventFilter",
```

Comment:
Properties should not have both `PrimitiveType` and `Type` defined and this is what is causing tests to fail.